packages/block-library/src/navigation-link/editor.scss

```diff
@@ -51,7 +51,7 @@
 		margin-left: 4px;
 	}
 
-	.block-editor-rich-text__editable.is-selected:not(.keep-placeholder-on-focus):not(:focus) [data-rich-text-placeholder]::after {
+	.block-editor-rich-text__editable.is-rich-text-selected:not(.keep-placeholder-on-focus):not(:focus) [data-rich-text-placeholder]::after {
```

Comment:
I have no idea what this is doing. Out of the scope of this PR though.

https://github.com/WordPress/gutenberg/pull/18374#pullrequestreview-313675133